homeassistant/components/izone/climate.py

```diff
@@ -314,12 +326,38 @@ def current_temperature(self) -> Optional[float]:
             return self._controller.temp_supply
         return self._controller.temp_return
 
+    @property
+    def control_zone_name(self):
+        """Return the zone that currently controls the AC unit."""
+        if not self._supported_features & SUPPORT_TARGET_TEMPERATURE:
```

Comment:
Invert this check and `return None` early. Then we can outdent below.

Reply:
Done. inverted.